plugin/vim-arsync.vim

```diff
@@ -124,4 +144,3 @@ command! ARsyncDown call ARsync('down')
 command! ARshowConf call ShowConf()
 
 autocmd VimEnter * call AutoSync()
-autocmd DirChanged * call AutoSync()
```

Comment:
This was a request from some user to ba able to change Dir while vim being open and still get syncing for the new dir.
i understand the bug, i have found a workaround by defining a group of autocmd and reseting each time you enter a new dir, so that you only get one sync going on.
I will fix it, later today.